corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -760,7 +763,6 @@ def _record_timing(self, status):
             else:
                 sync_log = self.restore_state.current_sync_log
                 sync_log_id = sync_log._id if sync_log else 'N/A'
-            log = logging.getLogger(__name__)
```

Comment:
I think this should not be removed. I'm forgetting why it's here, but in some contexts the global `logger` was not working in this function. I hope it doesn't turn out to be that way for your logging call above.